cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -367,11 +365,11 @@ parameter_types! {
 impl pallet_message_queue::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = weights::pallet_message_queue::WeightInfo<Runtime>;
-	#[cfg(feature = "runtime-benchmarks")]
+	#[cfg(all(not(feature = "std"), feature = "runtime-benchmarks"))]
```

Comment:
why introduce the additional filter on `feature = "std"`?

Reply:
When running `cargo test -p bridge-hub-rococo-runtime --features runtime-benchmarks`, the tests should rely on `BridgeHubMessageRouter` (because the tests check that the message is actually processed). However, when running the actual benchmarks, `BridgeHubMessageRouter` causes a wasm trap error. So it should only use `NoopMessageProcessor ` when running the benchmarks, not tests with the benchmark feature enabled. I agree this isn't an ideal fix though, I'll look into it further later

Reply:
I think the question was why to use `feature = "std"` instead of `test`.

Reply:
The `test` feature doesn't work. We've had the same issue with the fork versions for a local net and sepolia: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L592

`std` is enabled when running tests, so `test` would have ideal but it doesn't seem to work. [We added it for clarity](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L592) but it doesn't actually trigger when running tests.

Reply:
Anyways, please let me know if this is acceptable or not. 😅 If not I'll try the BenchmarkHelper way, it will just take a bit more time.

Reply:
To me it's acceptable if you add a code comment explaining above